app/main.py

```diff
@@ -1,3 +1,53 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | str) -> "Distance":
```

Comment:
The `__add__` method should accept `Distance` objects and numeric types, but the type hint `int | str` is incorrect. It should be `int | float | Distance` to match the task requirements .